tools/cold-backup/templates/patch.json

```diff
@@ -3,7 +3,7 @@
         "containers": [
             {
                 "name": "dummy",
-                "image": "centos:7",
+                "image": "quay.io/centos/centos:8",
```

Comment:
You should not change this. Last time I checked, the Quay repositories were not up to date.

Reply:
So, what image will we add here? Because everything from docker hub will not be pulled. @Frawless suggested Strimzi image. Or we can change it via `sed` command directly in ST

Reply:
Well, the base image also pulls from DockerHub. So if you cannot pull from there you should not even get to this test.

Reply:
After / if https://github.com/strimzi/proposals/pull/31 is approved, we can change it to UBI8 as well.

Reply:
Okay, so I'll revert it and we'll keep it fail until/if the proposal will be merged. Thanks @scholzj